dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java

```diff
@@ -25,42 +25,42 @@
 public class InaccessibleClassProblemTest {
   
   private ClassFile file;
-  private Symbol symbol;
-  
+  private ClassSymbol symbol;
```

Comment:
Is there a reason you have to change the type here?

Reply:
It's because InaccessibleClassProblem should take a class symbol, not a generic one.